api/lib/domain/usecases/assign-certification-officer-to-jury-session.js

```diff
@@ -25,6 +25,7 @@ module.exports = async function assignCertificationOfficerToJurySession({
   const certificationOfficerName = `${firstName} ${lastName}`;
 
   await finalizedSessionRepository.assignCertificationOfficer({
+    isPublishable: false,
     sessionId: integerSessionId,
     assignedCertificationOfficerName: certificationOfficerName,
   });
```

Comment:
```suggestion
  const certificationOfficerName = `${firstName} ${lastName}`;
  
  const finalizedSession = await finalizedSessionRepository.get({ sessionId: integerSessionId });
  finalizedSession.assignCertificationOfficer(certificationOfficerName);
  await finalizedSessionRepository.save(finalizedSession);
```

Puis on prendra soin d'écrire la méthode dans le modèle FinalizedSession qui va s'occuper, en plus d'assigner le nom de l'officer, de gérer les impacts métier de cette assignation (donc notamment passer le isPublishable à false dans le modèle directement)

Reply:
Yes mais là, la session resterait publiable si le isPublishable était déjà à true. On assignerait juste un membre du pôle certif :thinking: